File: PCP/App/ActivityMain.cs
using System;

using Android.App;
using Android.Content;
using Android.Runtime;
using Android.Views;
using Android.Widget;
using Android.OS;
using System.Net;
using System.Net.Sockets;
namespace App
{
    [Activity(Label = "PCP", MainLauncher = true, Icon = "@drawable/icon")]
    public class ActivityMain : Activity
    {
        
        UdpClient client = new UdpClient();
        IPEndPoint endpoint = new IPEndPoint(IPAddress.Parse("60.191.34.90"),10086);
        protected override void OnCreate(Bundle bundle)
        {
                
            base.OnCreate(bundle);
            //Fullscreen
            RequestWindowFeature(WindowFeatures.NoTitle);
            Window.SetFlags(WindowManagerFlags.Fullscreen, WindowManagerFlags.Fullscreen);
            // Set our view from the "main" layout resource
            SetContentView(Resource.Layout.Main);
            byte[] bytes= {0x23,0x23};
            client.Send(bytes, bytes.Length, endpoint);
            // Get our button from the layout resource,
            // and attach an event to it


        }
    }
}

File: app/src/com/example/PCP/ActivityMain.java
package com.example.PCP;

import android.app.Activity;
import android.os.Bundle;
import android.view.Window;
import android.view.WindowManager;

public class ActivityMain extends Activity {
    /**
     * Called when the activity is first created.
     */
    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        //设置全屏          放到setContentView(R.layout.main);上面
        this.requestWindowFeature(Window.FEATURE_NO_TITLE);
        this.getWindow().setFlags(WindowManager.LayoutParams.FLAG_FULLSCREEN,WindowManager.LayoutParams.FLAG_FULLSCREEN);
        setContentView(R.layout.main);
    }
}
